plugins/sinks/influxdb.go

```diff
@@ -0,0 +1,83 @@
+package main
+
+import (
+	"encoding/json"
+	api "github.com/emqx/kuiper/xstream/api"
+	_ "github.com/influxdata/influxdb1-client/v2"
+	client "github.com/influxdata/influxdb1-client/v2"
+	"log"
+	"time"
+)
+
+type influxSink struct {
+	url       string
+	table     string
+}
+
+type ListMap []map[string]float64
+
+func (m *influxSink) Configure(props map[string]interface{}) error {
+	return nil;
+}
+
+func (m *influxSink) Open(ctx api.StreamContext) (err error) {
+	logger := ctx.GetLogger()
+	logger.Debug("Opening influx sink")
+	return
+}
+
+func (m *influxSink) Collect(ctx api.StreamContext, data interface{}) error {
+	cli := connInflux();
+	logger := ctx.GetLogger()
+	if v, ok := data.([]byte); ok {
+		var out ListMap
+		json.Unmarshal([]byte(v), &out)
```

Comment:
The data is not always []map[string]float64. Other types need to be handled also. And the error of Unmarshall is not handled here.

Reply:
Thank you for your suggestion，I did some optimization of Influxdb.go 
Currently,only one type of data supported(Due to data format problems,is "[{}]" not "{}")
I will continue to optimize and solve this problem.

Reply:
ok. If other types are not supported currently, we'd better handle the error of unmarshall. For example, print out the error and return. Otherwise, it will crash the whole rule.